src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -0,0 +1,1497 @@
+                    OnPropertyChanged(nameof(WorkingDirectory));
+                }
+            }
+        }
+
+        public bool HasLaunchOption
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.LaunchBrowser;
+            }
+            set
+            {
+                if (SelectedDebugProfile != null && SelectedDebugProfile.LaunchBrowser != value)
+                {
+                    SelectedDebugProfile.LaunchBrowser = value;
+                    OnPropertyChanged(nameof(HasLaunchOption));
+                }
+            }
+        }
+
+        public bool IsExecutable
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.Executable;
+            }
+        }
+
+        public bool IsProject
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.Project;
+            }
+        }
+
+        public bool IsCommand
+        {
+            get
+            {
+                if (SelectedLaunchType == null)
+                {
+                    return false;
+                }
+
+                return SelectedLaunchType.Kind == ProfileKind.BuiltInCommand || SelectedLaunchType.Kind == ProfileKind.CustomizedCommand;
+            }
+        }
+
+        public bool IsProfileSelected
+        {
+            get
+            {
+                return SelectedDebugProfile != null;
+            }
+        }
+
+        public bool IsIISExpress
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.Kind == ProfileKind.IISExpress;
+            }
+        }
+
+        public bool IsIISOrIISExpress
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return SelectedDebugProfile.Kind == ProfileKind.IIS || SelectedDebugProfile.Kind == ProfileKind.IISExpress;
+            }
+        }
+
+        public bool IsWebProfile
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                return IsIISOrIISExpress;
+            }
+        }
+
+        public bool ShowVersionSelector
+        {
+            get
+            {
+                return !IsExecutable;
+            }
+        }
+        public bool IsCustomType
```

Comment:
>IsCustomType [](start = 20, length = 12)

This was used to know whether it could be deleted or not (I think). It is really a hold over from the DNX days when we reflected the list of commands from project.json in the debug UI (so you could run\debug and command).  I would just remove this